lib/ae_mdw/sync/async_tasks/migrate.ex

```diff
@@ -15,7 +15,7 @@ defmodule AeMdw.Sync.AsyncTasks.Migrate do
     Enum.each(migrations, fn %Migration{mutations_mfa: {module, mutations_fn, params}} ->
       module
       |> apply(mutations_fn, params)
-      |> AsyncStoreServer.write_mutations(done_fn)
+      |> AsyncStoreServer.write_mutations_without_async(done_fn)
```

Comment:
For a generic migration processing, this might cause errors on the sync server transaction because of the optimistic lock (it has happened in the past when the `AsyncStoreServer` used to persist data).

Reply:
I've now removed the async migration changes. I thought these mutations would run before the server started, my bad